rest/endpoints/user_recordings/profile_recordings.yml

```diff
@@ -381,6 +381,36 @@ paths:
                     type: array
                     items:
                       $ref: "https://raw.githubusercontent.com/booxmedialtd/api-documentation/master/rest/snippets/General/ForbiddenEntity.yml#/ForbiddenError"
+  # ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+  /most_popular/{period}:
+    get:
+      description: |
+        * **Available since:** BPLAT-12223
+      summary: Get most popular recordings
+      tags:
+        - Recordings
+      operationId: searchMostPopular
+      # TODO
+      parameters:
+        - in: path
+          name: period
+          required: true
+          description: Period to list recordings from. Valid options include "today", "next_week"
+          schema:
+            type: string
+      responses:
+        201:
+          description: "List of 100 most popular recordings"
+          content:
+            application/json:
+              schema:
+                type: "object"
+                required:
+                  - data
+                properties:
+                  data:
+                      $ref: "https://raw.githubusercontent.com/booxmedialtd/api-documentation/master/rest/snippets/EnrichedMetadata/BroadcastSlotEntity.yml#/BroadcastSlotEntity"
```

Comment:
I think it should be array of BroadcastSlotEntity, no?

Reply:
still now rendering correctly. For reference: https://github.com/booxmedialtd/api-documentation/blob/828b8e6aad95610b1c8e1e9daf0fed8a2eff46a7/rest/endpoints/user_recordings/profile_recordings.yml#L523-L532